index.html

```diff
@@ -152,8 +152,49 @@ <h3>Skills</h3>
         </ul>
       </li>
     </ul>
-
   </section>
+  <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+    <fieldset class="contact">
+      <h2 class="contactheader">Get Started</h2>
+      <p class="intro">Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      <label for="name"></label>
+      <input type="text" class="name" id="name" name="name" placeholder="Your Name" required />
+      <br>
+      <label for="name"></label>
+      <input type="text" class="name" id="email" name="email" value="Your Email" required />
+      <br>
+      <label for="message"></label>
+      <textarea name="message" class="message" id="message" placeholder="Enter your message here..."></textarea>
+      <input type="submit" class="btn1  " value="Start collaboration" />
+    </fieldset>
+  </form>
+  <footer>
+    <div  class="resume">
+      <p>Get my resume</p>
+    </div>
+    <ul class="social">
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_stackoverflow.png" alt="stackoverflow"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_github.png" alt="Github"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_twitter.png" alt="twitter"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_linkedin.png" alt="linkedin"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_medium.png" alt="Medium"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_angellist.png" alt="Angelist"></a>
```

Comment:
Great work so far. I only will suggest that the icons for the social media have a real link to your pages. That will avoid broken links and improve the UX.

Reply:
- [ ] Good job with this review and nice job adding the why👍👍 as stated earlier kindly add emoji's to make the review friendlier😉